src/private.rs

```diff
@@ -1,30 +1,42 @@
 use crate::Cookies;
 use cookie::{Cookie, Key};
 
-/// A child cookie jar that encrypts its cookies.
+/// A child cookie jar that provides authenticated encryption for its cookies.
+///
+/// A _private_ child jar signs and encrypts all the cookies added to it and
+/// verifies and decrypts cookies retrieved from it. Any cookies stored in a
+/// `PrivateJar` are simultaneously assured confidentiality, integrity, and
+/// authenticity. In other words, clients cannot discover nor tamper with the
+/// contents of a cookie, nor can they fabricate cookie data.
 pub struct PrivateCookies<'a> {
     cookies: Cookies,
     key: &'a Key,
 }
 
 impl<'a> PrivateCookies<'a> {
-    /// Creates an instance of `PrivateCookies` with parent `cookies` and key `key`. This method is
```

Comment:
The previous description was correct. Look at signature, we don't even have `parent` variable there